src/stories/Login.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import { Button } from './Button';
+import './Login.css';
+
+const Login = () => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleLogin = (e) => {
+    e.preventDefault();
+    console.log(`Logging in with username: ${username} and password: ${password}`);
```

Comment:
It's not the best solution to print the user's password in the console. Let's remove this line